wordpress-mobile/WordPress-iOS
# Improve upload on background

Related GB-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/576

This PR improves the use of thumbnails for image uploads to use the local cache images for Media. This is especially important when we have error states on background uploads.

To test:
 - Create a post in GB
 - Add an image to it
 - Publish while the upload is going on
 - Go back to post and see if the image shows up correctly
 - Repeat but simulate an error while the upload is going on.
